### ISSUE TITLE
Development Guide is missing instructions about using latest version

### ISSUE THREAD

**Issue by abitrolly** [pypa/setuptools#2532]
https://setuptools.readthedocs.io/en/latest/development/developer-guide.html

Installing from `git clone` the normal way fails.
```
$ pip install -e .
...
  RuntimeError: Cannot build setuptools without metadata. Run `bootstrap.py`.
```
The role of `bootstrap.py` is not very clear. It is not described in the docs.

**Comment by jaraco** [comment]
Agreed it's not clear. It's also something I'd like to be rid of rather than invest time cementing the approach. Unfortunately, being rid of it might require [Python packaging build support for editable packages](https://github.com/pypa/packaging-problems/issues/256).

In the meantime, my instinct is that the failure provides a clear directive on what to do, so while it causes a "fail first" experience, it's easily remedied.

I'd be open to contributions to the docs to provide more context that would have made your experience less surprising.

**Comment by jacalata** [comment]
This seems to be solved by PEP 660? If not, I can't repro the failure and bootstrap.py itself is gone, so presumably it has been addressed somehow.

**Comment by abravalheri** [comment]
I think the main problem is still the cyclic dependency on wheel.
